html/datasets.html

```diff
@@ -0,0 +1,313 @@
+                            }
+                        }
+                        html += "</small></span>";
+                    }
+
+                    // display distributions
+                    if (record["distributions"]) {
+                        html += "<br /><span><small>Distributions: ";
+                        if (record["distributions"].length == 0) {
+                            html += 'N/A'
+                        } else {
+                            html += "<br /> &emsp;&emsp;&emsp;&emsp;";
+                            for (var i = 0; i < record["distributions"].length; i++) {
+                                html += ("<a href=\"" + record["distributions"][i]["accessURL"] + "\" target=\"_blank\">"
+                                        + record["distributions"][i]["name"] + "</a>");
+                                html += " (Direct access: ";
+                                for (var j = 0; j < record["distributions"][i]["files"].length; j++){
+                                    html += ("<a href=\"" + record["distributions"][i]["files"][j]["downloadURL"]
+                                            + "\" target=\"_blank\">"
+                                            + record["distributions"][i]["files"][j]["name"] + "</a>");
+                                    if (j < record["distributions"][i]["files"].length - 1) {
+                                        html += "; ";
+                                    }
+                                }
+                                html += ")";
+                                if (i < record["distributions"].length - 1) {
+                                    html += "; <br /> &emsp;&emsp;&emsp;&emsp;";
+                                }
+                            }
+                        }
+                        html += "</small></span>";
+                    }
+
+                    // diaplay abstract or not ??
+
+                    // Badges
+
+                    html += "<br />";
+
+                    if (record["doi"]) {
+                        var escapedDOI = encodeURIComponent(record["doi"]);
+                        html += "<a href=\"" + doiUrl + "\" target=\"_blank\"><img style='vertical-align: middle' src='//dcotest.tw.rpi.edu/badges/badge?title=DOI&text="+escapedDOI+"&color=66CCFF'/></a>&emsp;";
+                    }
+
+                    var escapedDCOId= encodeURIComponent(record["dcoId"].substring(25));
+                    html += "<a href=\""+record["dcoId"]+"\" target=\"_blank\"><img style='vertical-align: middle' src='//dcotest.tw.rpi.edu/badges/badge?title=metadata&text="+escapedDCOId+"&color=D14719'/></a>";
+
+                    html += "</td></tr>";
+                    return html;
+                },
+                selected_filters_in_facet: true,
+                show_filter_field : true,
+                show_filter_logic: true,/*,
+                post_init_callback: function(options, context) {
```

Comment:
Just get rid of the code. We don't need it, can always replicate from the publication browser